data/network/src/commonMain/kotlin/social/androiddev/common/network/MastodonApi.kt

```diff
@@ -9,6 +9,7 @@
  */
 package social.androiddev.common.network
 
+import social.androiddev.common.network.model.Application
```

Comment:
@thebino I just noticed the GPL license header is missing from files in `network` module. Is this intentional or should I add it? If so - I'll do it in follow-up PR